grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -163,4 +165,150 @@ public static Value of(Map<Value, Value> fields) {
         .setCollection(Collection.newBuilder().addAllElements(values).build())
         .build();
   }
+
+  public static boolean bool(Value value) {
+    if (value.getInnerCase() != InnerCase.BOOLEAN) {
+      throw new IllegalArgumentException("Expected boolean value");
```

Comment:
should we enrich all exception messages with the value that was provided, it might help the user?

Reply:
good suggestion: https://github.com/stargate/stargate/pull/1305/commits/12fc903cccb14e4f78cae69e17dccac92eb4c73a